frontend/src/main/java/puf/frisbee/frontend/model/PlayerModel.java

```diff
@@ -23,12 +42,34 @@ public void setLoginStatus(boolean status) {
     }
 
     @Override
-    public boolean register(String name, String email, String password) {
+    public boolean register(String name, String email, String password) throws URISyntaxException, IOException, InterruptedException {
         this.name = name;
         this.email = email;
         this.password = password;
+        this.isLoggedIn = true;
+
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        String playerJson = objectMapper.writeValueAsString(this);
+
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(new URI("https://puf-frisbee-backend.herokuapp.com/api/players/register"))
+//                .uri(new URI("http://localhost:8080/api/players/register"))
```

Comment:
Weil ich hier grad reingelaufen bin - das wäre auch toll über Env variables, aber das seh ich auch in nem extra Ticket.

Reply:
Werde diese Env variables dann auch in einer eigenen Class/File zusammengefasst, so dass man sie dort zentral verwalten kann? Mir wäre sonst der Sinn nicht so ganz klar.

Reply:
Oder geht es dabei auch um Sicherheitsaspekte?